receiver/kafkareceiver/factory.go

```diff
@@ -80,6 +81,9 @@ func WithLogsUnmarshalers(logsUnmarshalers ...LogsUnmarshaler) FactoryOption {
 
 // NewFactory creates Kafka receiver factory.
 func NewFactory(options ...FactoryOption) component.ReceiverFactory {
+	// TODO: find a more appropriate way to get this done, as we are swallowing the error here
```

Comment:
We don't allow any TODOs in the codebase. Please open an issue if you'd like to track this.

Reply:
Deleted 👍